src/js/Content/Features/Community/ProfileHome/FProfileStatus.js

```diff
@@ -0,0 +1,109 @@
+import {HTML, Localization, SyncedStorage} from "../../../../modulesCore";
+import {Feature, RequestData, SteamId, User} from "../../../modulesContent";
+
+export default class FProfileStatus extends Feature {
+
+    checkPrerequisites() {
+        return !this.context.isPrivateProfile;
+    }
+
+    async apply() {
+        const statusNode = document.querySelector(".profile_in_game_header");
+        if (!statusNode) { return; }
+
+        const ingameNode = document.querySelector(".profile_in_game_name");
+        const profileSteamId = SteamId.getSteamId();
+        let lenderSteamId = null;
+
+        // The input needed to get appid only appears when logged in
+        const appidNode = document.querySelector("input[name=ingameAppID]");
+        const bInGame = ingameNode && appidNode;
+        if (bInGame) {
+            const appid = appidNode.value;
+            const game = ingameNode.textContent;
+            // FinGameStoreLink
+            HTML.inner(ingameNode,
+                `<a href="//store.steampowered.com/app/${appid}">
+                <span data-tooltip-text="${Localization.str.view_in_store}">${game}</span>
+            </a>`);
```

Comment:
Although they share some code, this is better placed in its own module

Reply:
The problem I was facing it that it became a race condition.

Reply:
Changed it to this: 
![image](https://user-images.githubusercontent.com/4411977/123492981-5c8ae680-d61b-11eb-9f04-99333e89bfa1.png)

Now it doesn't rely on the same node.